src/routers/index.ts

```diff
@@ -2,7 +2,7 @@ import { Router } from 'express';
 
 import authentication from '@/routers/authentication';
 import people from '@/routers/people';
-import transactions from '@/routers/transactions/transactions';
+import transactions from '@/routers/transactions/index';
```

Comment:
if you are importing an index file, you can just import without specifying index. Ex: `import transactions from '@/routers/transactions/';`

Reply:
Very cool, did not no